package.json

```diff
@@ -37,10 +37,12 @@
   "devDependencies": {
     "@babel/core": "7.1.6",
     "@babel/helper-plugin-test-runner": "7.1.0",
+    "@babel/parser": "^7.2.0",
```

Comment:
Let's not use the hat version here, can we specify the exact pinned version (7.2.0).

Reply:
done